src/app/shared/form/builder/parsers/row-parser.ts

```diff
@@ -1,62 +1,87 @@
-import { DYNAMIC_FORM_CONTROL_TYPE_ARRAY, DynamicFormGroupModelConfig } from '@ng-dynamic-forms/core';
+import { Injectable, Injector } from '@angular/core';
+import {
+  DYNAMIC_FORM_CONTROL_TYPE_ARRAY,
+  DynamicFormGroupModelConfig
+} from '@ng-dynamic-forms/core';
 import { uniqueId } from 'lodash';
 
 import { IntegrationSearchOptions } from '../../../../core/integration/models/integration-options.model';
-import { DYNAMIC_FORM_CONTROL_TYPE_RELATION_GROUP } from '../ds-dynamic-form-ui/models/relation-group/dynamic-relation-group.model';
-import { DynamicRowGroupModel } from '../ds-dynamic-form-ui/models/ds-dynamic-row-group-model';
 import { isEmpty } from '../../../empty.util';
-import { setLayout } from './parser.utils';
+import { DynamicRowGroupModel } from '../ds-dynamic-form-ui/models/ds-dynamic-row-group-model';
+import { DYNAMIC_FORM_CONTROL_TYPE_RELATION_GROUP } from '../ds-dynamic-form-ui/models/relation-group/dynamic-relation-group.model';
 import { FormFieldModel } from '../models/form-field.model';
-import { ParserType } from './parser-type';
-import { ParserOptions } from './parser-options';
+import {
+  CONFIG_DATA,
+  FieldParser,
+  INIT_FORM_VALUES,
+  PARSER_OPTIONS,
+  SUBMISSION_ID
+} from './field-parser';
 import { ParserFactory } from './parser-factory';
+import { ParserOptions } from './parser-options';
+import { ParserType } from './parser-type';
+import { setLayout } from './parser.utils';
 
 export const ROW_ID_PREFIX = 'df-row-group-config-';
 
+@Injectable({
+  providedIn: 'root'
+})
 export class RowParser {
-  protected authorityOptions: IntegrationSearchOptions;
-
-  constructor(protected rowData,
-              protected scopeUUID,
-              protected initFormValues: any,
-              protected submissionScope,
-              protected readOnly: boolean) {
-    this.authorityOptions = new IntegrationSearchOptions(scopeUUID);
+  constructor(private parentInjector: Injector) {
```

Comment:
Again, it'd be nice to add basic TypeDocs here while editing the class. I realize you didn't create this class though.